src/mmw/js/src/draw/views.js

```diff
@@ -44,7 +44,10 @@ function actOnLayer(datum) {
 
 function validateRwdShape(result) {
     var d = new $.Deferred();
-    if (result.watershed) {
+    if (result.watershed.features[0].geometry.type === 'MultiPolygon') {
+        var message = 'Unable to generate a polygon to analyze'
```

Comment:
Since this is user facing, let's update to something like "Unable to generate a valid watershed area at this location", to be more meaningful to them.